src/Helpers/i18n_helpers.php

```diff
@@ -83,9 +83,9 @@ function getLanguageLabelFromLocaleCode($code, $native = false)
     {
         if (class_exists(Locale::class)) {
             if ($native) {
-                return ucfirst(Locale::getDisplayLanguage($code, $code));
+                return ucfirst(Locale::getDisplayName($code, $code));
```

Comment:
I had a quick check and we can drop ucfirst here as getDisplayName is already properly capitalized. 

Also, you can remove the } else { as it is returned early.

Reply:
Same for the other call

Reply:
I think need } else { in reason, when need translate for not current language

Reply:
Sorry, I should have been more clear as I did not mean the content of the else but rather to have it return properly.

```
if (true) {
   return 'bar';
} else {
   return 'foo';
}
```

is the same as:

```
if (true) {
   return 'bar';
}

return 'foo';
```

Reply:
Ok, my fail :-( . Fixed this.

Reply:
In this function has problem when class Locale not exists. But i check again on next weekend and if he be, then i fix too.